src/core/electrostatics_magnetostatics/coulomb.cpp

```diff
@@ -106,11 +106,11 @@ double cutoff(const Utils::Vector3d &box_l) {
   switch (coulomb.method) {
   case COULOMB_MMM1D:
     return std::numeric_limits<double>::infinity();
+  case COULOMB_MMM2D:
+    return 1e-40;
```

Comment:
Please don't use a magic number, this should be `std::numeric_limits<double>::min()`